soroban-sdk/src/token.rs

```diff
@@ -80,7 +80,7 @@ pub use TokenClient as Client;
 /// There are no functions in the token interface for minting tokens. Minting is
 /// an administrative function that can differ significantly from one token to
 /// the next.
-#[contractspecfn(name = "StellarAssetSpec", export = false)]
+#[contractspecfn(name = "TokenSpec", export = false)]
```

Comment:
Why does token spec have `approve` and `burn_from` functions, but not `transfer_from`?

Reply:
In the `TokenSpec` trait:
- The `approve` function is on line 121.
- The `burn_from` function is on line 209.
- The `transfer_from` function is on line 168.

Reply:
Sorry, apparently I haven't looked at the full fille, that's embarrassing.